dss/config.py

```diff
@@ -183,26 +206,30 @@ def get_es_index_name(index_type: ESIndexType,
                           shape_descriptor: typing.Optional[str] = None
                           ) -> str:
         """
-        Returns the fully qualified name of an Elasticsearch index of documents for a given
-        replica of a given type and shape.
+        Returns the fully qualified name of an Elasticsearch index containing documents of the given type in the given
+        replica.
         """
-        assert isinstance(replica, Replica)
-
-        deployment_stage = os.environ["DSS_DEPLOYMENT_STAGE"]
-        index = f"dss-{deployment_stage}-{replica.name}-{index_type.name}"
-        if shape_descriptor:
-            index = f"{index}-{shape_descriptor}"
-        if Config._CURRENT_CONFIG == BucketConfig.TEST:
-            index = f"{index}.{IndexSuffix.name}"
-        return index
+        return Config._get_es_index_name(index_type, replica, shape_descriptor)
 
     @staticmethod
     def get_es_alias_name(index_type: ESIndexType, replica: "Replica") -> str:
-        """Returns the alias for indexes"""
+        """
+        Returns the alias for the set of Elasticsearch indexes containing documents of the given type in the given
+        replica.
+        """
+        return Config._get_es_index_name(index_type, replica, 'alias')
+
+    @classmethod
+    def _get_es_index_name(cls,
+                           index_type: ESIndexType,
+                           replica: 'Replica',
+                           suffix: typing.Optional[str] = None) -> str:
+        assert isinstance(replica, Replica)
```

Comment:
I would argue that this assertion is not useful.

Reply:
Will remove.